app/views/common_parameters/index.html.erb

```diff
@@ -3,22 +3,23 @@
 <% title_actions display_link_if_authorized(_("New Parameter"), hash_for_new_common_parameter_path),
                  documentation_button('4.2.3Parameters') %>
 
-<table class="table table-bordered table-striped table-two-pane table-fixed">
+
+<table class="table table-bordered table-striped table-two-pane table-hover">
   <thead>
     <tr>
       <th class="col-md-3"><%= sort :name, :as => s_("Parameter|Name") %></th>
-      <th class="col-md-8"><%= sort :value, :as =>  s_("Parameter|Value")%></th>
-      <th class="col-md-1"></th>
+      <th class="col-md-8" style='min-width: 60%;'><%= sort :value, :as =>  s_("Parameter|Value")%></th>
+      <th class="col-md-1"><%= _('Actions') %></th>
     </tr>
   </thead>
   <tbody>
     <% for common_parameter in @common_parameters %>
       <tr>
-        <td class="display-two-pane ellipsis"><%= link_to_if_authorized common_parameter, hash_for_edit_common_parameter_path(:id => common_parameter).merge(:auth_object => common_parameter, :authorizer => authorizer, :permission => 'edit_globals') %></td>
-        <td class="ellipsis"><%= common_parameter.safe_value %></td>
+        <td class="display-two-pane ellipsis"><%= link_to_if_authorized trunc_with_tooltip(common_parameter), hash_for_edit_common_parameter_path(:id => common_parameter).merge(:auth_object => common_parameter, :authorizer => authorizer, :permission => 'edit_globals') %></td>
+        <td class='ellipsis' style='overflow-wrap: break-word;'><%= trunc_with_tooltip(common_parameter.safe_value, 80) %></td>
```

Comment:
`trunc_with_tooltip` should not be used in fixed width table cells with `ellipsis` class. I am not sure why the `table-fixed` class was removed from this table?